src/Http/ServiceProvider.php

```diff
@@ -95,23 +94,25 @@ function (Container $container) {
     public function bootstrap(Container $container)
     {
         $option = get_option('paypalplus_shared_options');
-        $cachePayPalJsFiles = wc_string_to_bool($option['cache_paypal_js_files']);
-        if ($cachePayPalJsFiles) {
-            $cronScheduler = $container->get(CronScheduler::class);
+        $cachePayPalJsFiles = isset($option['cache_paypal_js_files']) ? $option['cache_paypal_js_files'] : false;
+        $cachedPayPalJsFiles = wc_string_to_bool($cachePayPalJsFiles);
+        if (!$cachedPayPalJsFiles) {
+            return;
+        }
+        $cronScheduler = $container->get(CronScheduler::class);
 
-            add_filter(
-                'cron_schedules',
-                function (array $schedules) use ($cronScheduler) {
-                    return $cronScheduler->addWeeklyRecurrence($schedules);
-                }
-            );
+        add_filter(
+            'cron_schedules',
+            function (array $schedules) use ($cronScheduler) {
+                return $cronScheduler->addWeeklyRecurrence($schedules);
+            }
+        );
```

Comment:
```suggestion
        add_filter(
            'cron_schedules',
            static function (array $schedules) use ($cronScheduler) {
                return $cronScheduler->addWeeklyRecurrence($schedules);
            }
        );
```

Just added `static`.